Apps/Playground/Win32/App.cpp

```diff
@@ -391,33 +395,39 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
         }
         case WM_MOUSEMOVE:
         {
-            if (nativeInput != nullptr)
+            if (nativeInput != nullptr && !IsTouchEvent(GetMessageExtraInfo()))
```

Comment:
Does this mean when a touch happens, we get both a `WM_MOUSEMOVE` and a `WM_POINTERDOWN`?

Reply:
I had originally included it because I noticed an odd occurrence in the Windows Simulator where a single WM_MOUSEMOVE was being fired when the touch cursor entered the screen area but generally, WM_MOUSEMOVE events should **not** occur with touch events.

Reply:
I feel like it's ultimately unnecessary.  I'm gonna remove this check because I think this may be more on the simulator than how it would actually work.